iphone/TitaniumKit/TitaniumKit/Sources/API/TiAnimation.m

```diff
@@ -232,6 +240,28 @@ - (TiPoint *)center
   return center;
 }
 
+- (void)setColor:(id)color_
+{
+  [color release];
+  color = [[TiUtils colorValue:color_] retain];
+}
+
+- (id)color
+{
+  return color.name;
+}
+
+- (void)setBackgroundColor:(id)backgroundColor_
+{
+  [backgroundColor release];
+  backgroundColor = [[TiUtils colorValue:backgroundColor_] retain];
+}
+
+- (id)backgroundColor
+{
+  return backgroundColor.name;
```

Comment:
shouldn't this be returning a `TiColor`, not the name? What if the backgroundColor is `nil`?